pep/app/styles/components/_ember-table.scss

```diff
@@ -0,0 +1,23 @@
+.table,
+.ember-table {
+    thead {
+        th {
+            border-top: 1px solid #fff;
+        }
+        tr:nth-child(2) > th {
+            border-top: 1px solid #e9e9e9;
+            &:first-child {
+                padding-left: 0.75rem !important;
+            }
+        }
+        tr:first-child th:first-child {
+            border-radius: 0 !important;
+        }
+    }
+    th:last-child,
+    td:last-child,
+    th:first-child,
+    td:first-child {
+        padding-left: 0;
+    }
+}
```

Comment:
i think you might not need a bunch of these styles, or any, now that the addon's sass styles should be imported. but if any of the color based styles are still needed, they should use existing variables, or create new ones which need to go in both _variables-theme.scss, and themes/tomato.scss for theming purposes.

Reply:
We need these due to the sub columns